pkg/kubectl/resize.go

```diff
@@ -39,17 +40,29 @@ type PreconditionError struct {
 	ActualValue   string
 }
 
-func (pe *PreconditionError) Error() string {
+func (pe PreconditionError) Error() string {
 	return fmt.Sprintf("Expected %s to be %s, was %s", pe.Precondition, pe.ExpectedValue, pe.ActualValue)
 }
 
+type ControllerUpdateError struct {
+	ResourceVersion string
+	ReplicaCount    int
```

Comment:
If you keep this field, please change the name so it's clear if it's from spec or status and whether it's the value currently in the object or the value we're trying to set it to.